app/controllers/state_file/questions/nc_taxes_owed_controller.rb

```diff
@@ -0,0 +1,22 @@
+module StateFile
+  module Questions
+    class NcTaxesOwedController < QuestionsController
```

Comment:
same as above [dust]
```suggestion
    class NcTaxesOwedController < TaxesOwedController
```